packages/core/src/electron-browser/menu/electron-menu-contribution.ts

```diff
@@ -224,4 +238,15 @@ export class ElectronMenuContribution implements FrontendApplicationContribution
         });
     }
 
+    /**
+     * Returns the zoom level rounded to the nearest multiple of a specified value.
+     *
+     * @param zoomLevel the current window zoom level.
+     * @param nearestMultiple the nearest multiple to round to.
+     * @param isIncreasing true, if rounding before increasing zoom; false, if rounding before decreasing zoom.
+     */
+    private roundZoomLevel(zoomLevel: number, nearestMultiple: number, isIncreasing: boolean): number {
+        return isIncreasing ? Math.floor(zoomLevel / nearestMultiple) * nearestMultiple : Math.ceil(zoomLevel / nearestMultiple) * nearestMultiple;
```

Comment:
Presumably this logic is so that, when starting with a level that is not a multiple of 0.5, it will always move by at least 0.5 but by less than 1.0.  It would be nice if the reasoning behind this up/down rounding was explained instead of the boilerplate function documentation you currently have.

In fact looking at this further, it seems to me it would be simpler if this function was in-lined into the two places that call it.  Then you don't need the flag at all.